modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Step.scala

```diff
@@ -8,10 +8,12 @@ import Scalaz._
 /**
   * A list of `Executions` grouped by observation.
   */
-case class Step[+A](id: Int, config: StepConfig, executions: List[List[A]])
+case class Step[+A](id: Step.Id, config: StepConfig, breakpoint: Boolean, executions: List[List[A]])
 
 object Step {
 
+  type Id = Int
```

Comment:
For the time being the id is generated internally using the step position. If the sequence is reloaded its internal representation is regenerated from scratch. Therefore, the breakpoints are lost.